Components/image/image.css

```diff
@@ -0,0 +1,34 @@
+@import url('../../globalVar.css');
+.img-container{
+    align-items: center;
+    column-gap: 1rem;
+    display: flex;
+    justify-content: center;
+    margin: 1rem;
+}
+
+.img-responsive{
+    height: auto;
+    width: 60%;
+}
+
+.img-round{
+    border-radius: 50%;
+}
+
+@media only screen and (max-width : 768px){
```

Comment:
Try to follow mobile first approach. Go with min-width:768px

Reply:
As far as I remember, it was mentioned that both approaches are fine. If 80% users are using the desktop version, we have to design desktop first and similarly for mobile first approach.